tests/fixtures/transformer_test_data.py

```diff
@@ -999,6 +999,33 @@ def has_cropped_properly(self):
         focal_points=[],
         crop_left=None, crop_top=None, crop_right=None, crop_bottom=None,
         fit_in=True, adaptive=True, full=True
-    ), (400, 200, 1)),
+    ), (400, 200, True)),
+
+    (TestData(
+        source_width=500, source_height=500,
+        target_width=250, target_height="orig",
+        halign="middle", valign="middle",
+        focal_points=[],
+        crop_left=None, crop_top=None, crop_right=None, crop_bottom=None,
+        fit_in=True
+    ), (250, 250, True)),
+
+    (TestData(
+        source_width=500, source_height=500,
+        target_width="orig", target_height=250,
+        halign="middle", valign="middle",
+        focal_points=[],
+        crop_left=None, crop_top=None, crop_right=None, crop_bottom=None,
+        fit_in=True
+    ), (250, 250, True)),
+
+    (TestData(
+        source_width=500, source_height=500,
+        target_width="orig", target_height="orig",
```

Comment:
added cases to catch https://github.com/thumbor/thumbor/pull/877#discussion_r144110881